src/agoradatatools/etl/transform/apply.py

```diff
@@ -0,0 +1,44 @@
+from agoradatatools.etl.transform.custom import *
```

Comment:
Looking at this makes me wonder if there could be a `BaseTransform()` class that people extend from to create custom transformations?  It would definitely be a separate ticket to look into this.

This function could then look for classes that extend the base class and loop through it for processing. (Just an idea)

Reply:
Yeah, I was thinking about something similar. `apply_custom_transformations` definitely needs to be reworked so it doesn't have 9 `if`'s

Reply:
Simplifying this would be great, having to remember to manually add a new if statement for every dataset that needs a custom transform is annoying. I think another option might be to move the dataset-custom transform mappings out of this method and into the config file. I created AG-1092 to track this, feel free to add any thoughts.

Reply:
Prior to changing this logic, I think its key to have the tests or else we are at risk of unintentionally breaking something.